### PR TITLE
Packer: Install Xcode For Intel Macs.

### LINKED ISSUE #3720: Xcode 15.2 on test orka nodes
Xcode15.2 on test orka nodes  https://ci.adoptium.net/label/ci.role.test&&sw.os.osx&&orka&&hw.arch.x86

Details:

Reproducible comparison testing on Mac https://github.com/adoptium/temurin-build/pull/3844 requires Xcode15.2. However the test orka nodes doesn't have  xcode15.2 installed https://ci.adoptium.net/view/Test_grinder/job/Grinder/10771/console @steelhead31 also confirmed that no xcode15.2 installed on https://ci.adoptium.net/label/ci.role.test&&sw.os.osx&&orka&&hw.arch.x86

Orka build nodes does have Xcode15.2 installed https://ci.adoptium.net/label/xcode15.0.1&&build&&mac&&x64/ ( lable xcode15.0.1  should be changed to 15.0.2 https://github.com/adoptium/temurin-build/pull/3844#issuecomment-2178754616)

Asking to update orka test node template to have xcode 15.0.2 installed.

### PR DESCRIPTION
**Pull request by steelhead31**
Fixes #3720 

Intel Macs now require Xcode for reproducible build tests.

##### Checklist

- [X] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [X] VPC/QPC not applicable for this PR

### CODE REVIEW

File: ansible/packer/orka-base.pkr.hcl [hunk 1]
```diff
@@ -88,7 +88,6 @@ EOF
   # Install Xcode
   provisioner "ansible-local" {
     # We only install Xcode on the arm64 VM (build tools is enough for the Intel test VMs)
```
Comment (gdams) [inline ansible/packer/orka-base.pkr.hcl:90]:
```suggestion
```

**Comment by gdams** [comment]
Can we perhaps skip the install of Xcode 11.7 on intel to speed things up?  You'd need to make a change to the playbook to skip it on intel

**Comment by gdams** [comment]
Added a comment to https://github.com/adoptium/infrastructure/issues/3720 but I do wander if we'd be better off just running the reproducible tests on M1 hosts? Rosetta should be able to cope with this and it avoids us adding a massive package to the base image

**Comment by gdams** [comment]
Also as things stand this will break the github action as we don't pass the XCode SAS token to the job https://github.com/adoptium/infrastructure/blob/master/.github/workflows/packer.yml#L44

**Comment by steelhead31** [comment]
I'll put this back into draft for the moment, as Im away now for the next 10 days, and it needs some further work & discussion.

**Pull request closed without merging**